operate/services/service.py

```diff
@@ -715,21 +716,19 @@ def delete(self) -> None:
 class Service(LocalResource):
     """Service class."""
 
+    name: str
     version: int
     service_config_id: str
+    path: Path
+    package_path: Path
     hash: str
     hash_history: t.Dict[int, str]
+    agent_release: AgentRelease
     agent_addresses: t.List[str]
     home_chain: str
     chain_configs: ChainConfigs
     description: str
     env_variables: EnvVariables
-    agent_release: t.Dict
-
-    path: Path
-    package_path: Path
-
-    name: t.Optional[str] = None
```

Comment:
This is so that the fields are saved in the service config.json in a more scannable order.